src/components/Sum.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const Sum = (props) => {
```

Comment:
It's better to use object destructuring here
```suggestion
const Sum = ({a, b}) => {
```